homeassistant/components/romy/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""ROMY Integration."""
+
+import romy
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, LOGGER, PLATFORMS
+from .coordinator import RomyVacuumCoordinator
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """TODO."""
+
+    password = ""
+    if CONF_PASSWORD in config_entry.data:
+        password = config_entry.data[CONF_PASSWORD]
+
+    new_romy = await romy.create_romy(config_entry.data[CONF_HOST], password)
+
+    name = config_entry.data[CONF_NAME]
+    if name != new_romy.name:
+        await new_romy.set_name(name)
+        LOGGER.info("Settings ROMY's name to: %s", new_romy.name)
```

Comment:
Is this really wanted?

Reply:
you mean printing the new name? was more kind of debug information -> removed